templates/base.html

```diff
@@ -51,7 +51,7 @@
               <a class="nav-link" href="/global">Global</a>
             </li>
             <li class="nav-item">
-              <a class="nav-link" href="/Mine">Mine</a>
+              <a class="nav-link" href="/login">Mine</a>
```

Comment:
fixed navBar to send User from Mine tab to Login Page before entering his Mine Page

Reply:
after the login, the user need to get in his page (with his house_id)